src/libmongoc/tests/test-mongoc-cursor.c

```diff
@@ -435,7 +435,7 @@ _make_cmd_deprecated_cursor (mongoc_collection_t *coll)
                                      0,
                                      0,
                                      0,
-                                     tmp_bson ("{'ping': 1}"),
+                                     tmp_bson ("{'isMaster': 1}"),
```

Comment:
The C driver has a deprecated command cursor, which returns the command reply as the first (and only) document returned in the cursor. 

The cursor tests here (under "/Cursor/common/*") test common behavior across all cursor types. In the process, they send arguments alongside the command like `batchSize`. Since `ping` no longer ignores extra arguments, this changes to use `isMaster`.